target/linux/pistachio/base-files/etc/asound.conf

```diff
@@ -0,0 +1,17 @@
+pcm.!default {
```

Comment:
Doesn't this file belong to the asound package?

Then you can have install script similarly like we have in uboot-env.